requirements.txt

```diff
@@ -2,3 +2,4 @@ pyyaml
 gitpython
 unflatten
 pandas
+xlrd
```

Comment:
I think this might need to be  `xlrd==1.2.0` following on from a comment (which I can't seem to find now) re. xlrd now only being able to read .xls files